debian/autobake-deb.sh

```diff
@@ -91,6 +94,11 @@ case "${CODENAME}" in
     exit 1
 esac
 
+if [ -n "${SALSA_PREP:-}" ]
```

Comment:
There could be some comment why this is here as I don't understood reasoning? If we are not in Salsa exit?

Reply:
`:-` to give it a default value in case its not set. Will prevent failures in bash strict `+x` mode and gets recognised by shellcheck.

Reply:
Please add inline comment on what this section is, what uses it and why. Anybody can look up the syntax on a man page but it is hard to guess why you have this seemingly unnecessary logic here.

Reply:
Ok, so there should be an inline comment saying that the section is tailored for use with https://github.com/MariaDB/mariadb.org-tools/blob/master/buildbot.mariadb.org/ci_build_images/debian.Dockerfile to make Docker images with preinstalled build-dependencies for each Debian/Ubuntu release, and thus you want to generate the `debian/control` files but not run autobake.